rhel/SOURCES/grafana-5.1.3-change-icon.patch

```diff
@@ -0,0 +1,45 @@
+diff --git a/public/img/percona-favicon.ico b/public/img/percona-favicon.ico
+new file mode 100644
+index 0000000000000000000000000000000000000000..f426064d6f505e77365a8980046b58ae465ea6b0
+GIT binary patch
+literal 894
+zcma)*y-QnR6o=2f1!I;Hp`@aNT`C2!h#-ZEi~0xX;8Hu38qlQ<ii;p9IEZvnhb|pd
+z5C;dpQ2H?>*ox_(f?^{iiV^YKL1IHplY4#M`^rTkf;YeS@;mQ&&U<do<%$^md_Lh`
+zC~Z$f{37y-#g@Easr^S@W0S|_)vNx}FsiIw@95cmsBrP;xoum`pQ2(0je_4uyi0gb
+z;jjBu^$i<ME!(zjw{|LCk&bQ)%^+E~k!HASp4!2oeT31x>SYFXmp^NlI@J9BQIe;k
+z(7CLs*^{Tm%esmOqlG%$X{~wSukq}gkcBW*F#!N|rgw=JOye#;bCk`Wvs@9{nXJn&
+zG3KUg_4yxH{sTW-un~rkj_S`kEr5%v){K&{_`^(+RM%$4oWA8TgglZOvOf$4T=hO?
+zRIH<yjge&PI-1D7{;o<8hVWl9O-eI0fmr{<saf||3(si>$;{_SdMJ{{5as}k=y@QX
+z_io42yoUznE;!H`sf8ibDehqUl@0$$7;OjM_CtiJ=s)`~et|G|_rHV^?kn#8c4_vU
+zdHc?_@HruX3u`m6bzL@kl|PazHc92cZb*umznxDZ%zZLdBC_WZ3Gv<Z8q}%vz@_qP
+JDOS~}+8cb5Xo~;<
+
+literal 0
+HcmV?d00001
+
```

Comment:
Please remove this binary part of patch

Reply:
My suggestion is: you can place `percona-favicon.ico` file into `pmm-server-packaging/rhel/SOURCES/percona-favicon.ico`
and than move it public/img/percona-favicon.ico
@delgod please confirm if this appropriate way

Reply:
@askomorokhov  totally agree